packages/stencil-library/src/components/button/button.tsx

```diff
@@ -25,57 +21,87 @@ export class Button {
                     return "usa-button--outline"
                 case 'link':
                     return "usa-button--unstyled"
-                case 'danger':
-                    return "usa-button--secondary"
             }
         }
 
         const getDarkModeVariantClassName = (variant: ButtonVariant) => {
             switch (variant) {
+                case 'primary':
+                    return "primary-button-dark"
                 case "secondary":
                     return "usa-button--outline usa-button--inverse"
                 case 'link':
-                    return "usa-button--unstyled usa-button--outline usa-button--inverse"
+                    return "usa-button--unstyled unstyled-button-dark"
+                default:
+                    return 'primary-button-dark'
```

Comment:
[dust] I think it might be a good idea to add "usa" or "nj" as a prefix to these custom classes in order to avoid styling naming conflicts in projects. 

Maybe we can create an issue in the repo to track for later?

Reply:
I think issue will be good here, we should align this across the njwds repo as well